src/art-styles/circle_sketch.ts

```diff
@@ -0,0 +1,480 @@
+// fill scribble effect
+// draw squares first
+
+import p5 from "p5";
+import p5Types, { Color } from "p5";
+import GenericSketch from "./generic_sketch";
+import Scribble from "./scribble";
+
+export interface CircleSketchOptions {
+  numOfBoxes: number;
+  paletteIndex: number;
+  noFill: boolean;
+  removeBlocks: number;
+  animate: boolean;
+  paperIndex: number;
+}
+
+// const paper_links: Array<string> = [
+//   "https://maroon-petite-shrew-493.mypinata.cloud/ipfs/QmYPmQdJQJLtvMpbXgeHjWzatnyVyCMxcpQsknR8AbYuR6",
+//   "https://maroon-petite-shrew-493.mypinata.cloud/ipfs/QmXdCnGST3VXiBBGAtEGLu4h3WyjZhayLJD9d6cUYtdDaj",
+//   "https://maroon-petite-shrew-493.mypinata.cloud/ipfs/Qmc1UUU1dkfnKQNAtuZEq5QEDNpwAj4DR5N8eP2VFVxxgk",
+// ];
+
+const paper_links: Array<string> = [
+  "https://blockalizer-animation-template.s3.us-east-2.amazonaws.com/paper1.webp",
+  "https://blockalizer-animation-template.s3.us-east-2.amazonaws.com/paper2.webp",
+  "https://blockalizer-animation-template.s3.us-east-2.amazonaws.com/paper3.webp",
+];
+
+export class CircleSketch extends GenericSketch {
```

Comment:
This file contains a whole bunch of things that I think you aren't using.

Reply:
yes will clean up we aren't using